structuredefinitions/UKCore-RequestGroup.xml

```diff
@@ -0,0 +1,190 @@
+			<path value="RequestGroup.author.identifier.assigner"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization"/>
+			</type>
+		</element>
+		<element id="RequestGroup.reasonReference">
+			<path value="RequestGroup.reasonReference"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Condition"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-DiagnosticReport"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-DocumentReference"/>
+			</type>
+		</element>
+		<element id="RequestGroup.reasonReference.identifier.assigner">
+			<path value="RequestGroup.reasonReference.identifier.assigner"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization"/>
+			</type>
+		</element>
+		<element id="RequestGroup.note.author[x]">
+			<path value="RequestGroup.note.author[x]"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Patient"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-RelatedPerson"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization"/>
+			</type>
+		</element>
+		<element id="RequestGroup.action.participant">
+			<path value="RequestGroup.action.participant"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Patient"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-PractitionerRole"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-RelatedPerson"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Device"/>
+			</type>
+		</element>
+		<element id="RequestGroup.action.participant.identifier.assigner">
+			<path value="RequestGroup.action.participant.identifier.assigner"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization"/>
+			</type>
+		</element>
+		<element id="RequestGroup.action.resource">
+			<path value="RequestGroup.action.resource"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationRequest"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationAdministration"/>
+				<targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationDispense"/>
```

Comment:
Did we decide to also add UKCore-ServiceRequest here?

Reply:
That should have gone back to default as a reference (any)